lib/v2/datasource/remote/firebase/firebase_push_notification_service.dart

```diff
@@ -25,8 +24,18 @@ class PushNotificationService {
           print("FirebaseMessaging getInitialMessage: ${message}");
         }
       });
+      await _firebaseMessaging.subscribeToTopic("PUSH_RC");
+
+      FirebaseMessaging.onMessage.listen((RemoteMessage message) {
+        print("FirebaseMessaging onMessage -> " + message.data.toString());
```

Comment:
This instruction onMessage below is no longer necessary, you already defined it here.

Reply:
thanks, fixed